roles/development/tasks/docker.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Hi, @wakawakaxin 

we need to setup the docker to satisfy different environments. Please check the right environments configuration.